routers/api/v1/api.go

```diff
@@ -937,6 +937,10 @@ func Routes(sessioner func(http.Handler) http.Handler) *web.Route {
 				m.Group("/git", func() {
 					m.Group("/commits", func() {
 						m.Get("/{sha}", repo.GetSingleCommit)
+						m.Get("/{sha}.diff",
```

Comment:
```suggestion
						m.Get("/{sha}.{diffType:diff|patch}",
```
can we unify that function directly within the router - so we only have one new endpoint

Reply:
I'll look into this, just wait a moment 🙃

Reply:
I merged them and it works... But `make generate-swagger` doesn't work. It always returns (the swagger JSON in the commit is something I tried to resolve it, but it's not that what it should be-9:
```
operation (repoDownloadCommitDiffOrPatch): yaml: line 21: did not find expected key
exit status 1
```

Reply:
That's also the reason why the CI fails

Reply:
use space not tabs

Reply:
Thanks for solving this :)